core/client/fs/src/main/java/alluxio/client/file/cache/PageNotFoundException.java

```diff
@@ -10,7 +10,7 @@
  *
  */
 
-package alluxio.client.file.cache.store;
+package alluxio.client.file.cache;
```

Comment:
we typically put all Alluxio exceptions together rather than leaving them  in different packages.

Reply:
Moved.